lib/service_actor/collectionable.rb

```diff
@@ -6,23 +6,42 @@
 # Example:
 #
 #   class Pay < Actor
-#     input :provider, in: ["MANGOPAY", "PayPal", "Stripe"]
+#     input :provider, inclusion: ["MANGOPAY", "PayPal", "Stripe"]
```

Comment:
I like the new name, but we should keep supporting `in`, at least for the "simple" syntax.

Reply:
I’ve started soft-deprecating `in` by supporting both syntaxes in:  https://github.com/sunny/actor/pull/84